src/main/java/org/broadinstitute/consent/http/db/SamDAO.java

```diff
@@ -33,7 +33,7 @@
 import org.checkerframework.checker.nullness.qual.NonNull;
 import org.checkerframework.checker.nullness.qual.Nullable;
 
-public class SamDAO implements ConsentLogger {
+public class SamDAO implements ConsentLogger, SamDefaults {
```

Comment:
In general I'd recommend using embedding over inheritance, as it's more flexible and doesn't necessarily change an object's external interface. So instead of `implements SamDefaults` I'd add a `SamDefaults` member which is initialized in the constructor.

Reply:
Let me look at that approach. I tend to use inheritance for trait-like behavior, which is what I'm doing in this case.